server/icat/include/icatHighLevelRoutines.hpp

```diff
@@ -288,4 +288,15 @@ auto chl_update_ticket_write_byte_count(RsComm& _comm, const rodsLong_t _data_id
 /// \since 4.2.12
 auto chl_get_delay_rule_info(RsComm& _comm, const char* _rule_id, std::vector<std::string>* _info) -> int;
 
+/// \brief High-level wrapper for atomically updating rows in R_DATA_MAIN for all replicas of a particular data object.
+///
+/// \param[in,out] _comm iRODS comm structure
```

Comment:
in,out -> in?

Reply:
I've never been clear on how to label these when they're non-const references or pointers to non-const types. I guess we're conveying intent, but not guaranteeing that non-const parameters won't be modified?

Reply:
Right. I think expressing intent is the best path forward here.

Reply:
Done